.github_changelog_generator

```diff
@@ -0,0 +1,14 @@
+user=cms-gem-daq-project
+project=gem-plotting-tools
+bugs-label="**Fixed known bugs:**"
+enhancement-label="**Implemented enhancements:**"
+issues-label="**Closed issues:**"
+header-label="# Change Log ```gempython_gemplotting```"
+# front-matter [JSON]        Add YAML front matter. Formatted as JSON because it's easier to add on the command line
+include-labels=""
+exclude-labels="duplicate,answer,question,'Type: Question','Type: Duplicate','Type: Answer'"
+bug-labels="bugfix,bug,Bug,'Type: Bug','Type: Bugfix'"
+enhancement-labels="enhancement,Enhancement,'Type: Enhancement'"
+issue-line-labels=""
+release-branch=master
```

Comment:
Does this mean we always build from master?

Reply:
As written, releases come from `master`, such that the changelog will always make this assumption when pulling issues/PRs to parse and fill the per-release changelog
This aspect needs more work, as I want a full changelog, per release changelog, and release notes, each possibly serving a different purpose and presenting info in a slightly different way

Reply:
Will documentation for how to make a release be provided?

Also since releases will now come from `master` does this mean we are no longer following the `git-flow` described [here](http://nvie.com/posts/a-successful-git-branching-model/)?  If so will a new model be provided (a diagram like the one in that webpage I think is all that is necessary).

Reply:
Sorry, "tagged" releases are always on `master`

Reply:
Got it, so "release" still comes from `develop` but a "tag" comes from `master`

Here tags are displayed in the output of `git tag`, e.g.:

```
cmsgemos% git tag
v0.1.0
v0.2.0
v0.2.1
v0.2.2
v0.2.3
v0.3.0
v0.3.1
```

Whereas a release is shown in the release url, e.g. https://github.com/cms-gem-daq-project/gem-plotting-tools/releases

Do I understand this correctly...?

Reply:
* A `github` "release", is a tag on `master` with added files for installation
* A "release" branch is a branch (initially from `develop`) that targets a "release" (by incorporating some decided upon additions, features, etc.) and eventually tracks that "release" with bugfixes